jobs/integration/validation.py

```diff
@@ -1246,7 +1246,6 @@ async def test_audit_custom_policy(model, tools):
 
 
 @pytest.mark.asyncio
-@pytest.mark.skip_arch(["aarch64", "s390x"])
```

Comment:
FYI, i added a multiarch nginx image to rocks, so we don't need to skip this test anymore.